.golangci.yml

```diff
@@ -15,6 +15,8 @@ linters-settings:
       - standard
       - default
       - prefix(github.com/lxc/incus)
+  goimports:
```

Comment:
As mentioned in https://github.com/lxc/terraform-provider-incus/pull/168/files#r1849628258, having `goimports` and `gci` at the same time seems redundant to me.